Van_Marukyan/python/fibonacci/fibonacci.py

```diff
@@ -0,0 +1,13 @@
+def Fibonacci(n):
+    if n < 0:
+        return("Incorrect input")
+    elif n == 0:
+        return(1)
+    elif n == 1:
+        return(1)
+    elif n ==2:
+        return(1)
+    else:
+        return(Fibonacci(n - 1) + Fibonacci(n - 2))
+
+print(Fibonacci(7))
```

Comment:
- No any user input support
- No validation for argument as string
![image](https://user-images.githubusercontent.com/80816144/119726978-ced4a380-be82-11eb-994d-a6eb7d3a53e2.png)
- For 0 the values should be 0 like in wikipedia
https://en.wikipedia.org/wiki/Fibonacci_number
- Fix style as in pep8